### PR TITLE
Add 5 cases on view blueprint page.

### PR DESCRIPTION
**Pull request by henrywang**
1. should show correct blueprint description title
2. should show correct blueprint name under Details tab
3. should show correct blueprint description under Details tab
4. should update description
5. should not update description if click cancel button

### CODE REVIEW

File: test/end-to-end/package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "test": "jest --runInBand --forceExit"
   },
   "dependencies": {
+    "faker": "^4.1.0",
```
Comment (atodorov) [inline test/end-to-end/package.json]: Why do we need this?

**Comment by atodorov** [comment]
@martinpitt can you review?

File: test/end-to-end/package.json (hunk 1 above)
Comment (henrywang) [reply test/end-to-end/package.json]: We need to generate a random blueprint description sentence to update for every test run. That'll avoid some cache issue which will impact string match with cached value but not new value.
faker is a good tool to generate all kinds of random things, like word, sentence, etc.

Comment (atodorov) [reply test/end-to-end/package.json]: In this case please update the commit log to describe why the commit adds a new dependency.

Comment (henrywang) [reply test/end-to-end/package.json]: OK, Thanks.

**Pull request merged**